components/src/core/components/Buzz/BuzzPostInput.vue

```diff
@@ -0,0 +1,163 @@
+<!--
+/*
+ * This file is part of OrangeHRM Inc
+ *
+ * Copyright (C) 2020 onwards OrangeHRM Inc
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see  http://www.gnu.org/licenses
+ */
+-->
+
+<template>
+  <div :class="classes">
+    <textarea
+      rows="1"
+      ref="textArea"
+      class="oxd-buzz-post-input"
+      :disabled="disabled"
+      :value="modelValue"
+      @blur="onBlur"
+      @focus="onFocus"
+      @input="onInput"
+      v-bind="$attrs"
+    />
+    <div class="oxd-buzz-post-slot">
+      <slot></slot>
+    </div>
+  </div>
+  <oxd-text v-if="hasError" tag="span" class="oxd-input-field-error-message">
+    {{ message }}
+  </oxd-text>
+</template>
+
+<script lang="ts">
+import {
+  ref,
+  toRef,
+  nextTick,
+  PropType,
+  computed,
+  onMounted,
+  defineComponent,
+} from 'vue';
+import {Rule} from '../../../composables/types';
+import useField from '../../../composables/useField';
+import Text from '@ohrm/oxd/core/components/Text/Text.vue';
+
+interface TextAreaInputEvent extends InputEvent {
+  target: HTMLTextAreaElement;
+}
+
+export default defineComponent({
+  name: 'oxd-buzz-post-input',
```

Comment:
Any possibility to request a name for this component, like _expandable input_ or something,
https://oxd-prod-infinity.orangehrm.com/pages/comment-box.html

Reply:
This is exclusively used in buzz, that's why i used as this.